app/views/bulk_actions/catkey_and_barcode_csv_jobs/_new.html.erb

```diff
@@ -10,6 +10,7 @@
     <ul>
       <li>Druid (required)</li>
       <li>Catkey (optional). May be repeated if an item is associated with multiple catkeys.</li>
+      <li>Refresh (optional). Set to "true" to indicate first catkey is refreshable; set to "false" to indicate first catkey is not refreshable.</li>
```

Comment:
"... to indicate first catkey is refreshable; set to "false" to indicate *no* catkey is ~~not~~ refreshable" for your consideration.

Also, it's not the catkey that is refreshable, it's the descriptive metadata that is refreshable from the source represented by the catkey ...

"... to indicate descriptive metadata is refreshable from the first catkey ..." ?

Reply:
Updated.